pointerevents/pointerevent_boundary_events_at_implicit_release_hoverable_pointers-manual.html

```diff
@@ -42,12 +42,10 @@
             detected_pointertypes[event.pointerType] = true;
             event_log.push(event.type + '@' + event.target.id);
             if (event.type == 'lostpointercapture') {
-              setTimeout(function() {
-                test_pointer_event.step(function () {
+              test_pointer_event.step_timeout(function () {
                   var expected_events = "pointerup, lostpointercapture, pointerout, pointerleave";
```

Comment:
I think we should then adjust the indentation in 46 and 47.

Reply:
good catch. fixed.